File: python/roman_numerals_spec.py
from roman_numerals import to_roman

print(to_roman(1) == 'I')
print(to_roman(3) == 'III')
print(to_roman(4) == 'IV')

# add tests to cover different edge cases

print(to_roman(99) == 'XCIX')
print(to_roman(949) == 'CMXLIX')
print(to_roman(888) == 'DCCCLXXXVIII')

# output:
# True
# True
# True
# True
# True
# True

File: js/romanNumeralsSpec.js
let rn = require("./romanNumerals.js");

console.log(rn.toRoman(1) === 'I');
console.log(rn.toRoman(3) === 'III');
console.log(rn.toRoman(4) === 'IV');
console.log(rn.toRoman(99) == 'XCIX')
console.log(rn.toRoman(949) == 'CMXLIX')
console.log(rn.toRoman(888) == 'DCCCLXXXVIII')

// output: 
// true
// true
// true
// true
// true
// true